src/vs/workbench/contrib/debug/common/debug.ts

```diff
@@ -629,6 +629,7 @@ export interface IDebuggerContribution extends IPlatformSpecificAdapterContribut
 
 	// supported languages
 	languages?: string[];
+	enableBreakpointsFor?: { languageIds: string[] };
```

Comment:
I suggest to make languageIds optional, so that you can legally check for its existence.